src/auto/injector.js

```diff
@@ -849,7 +853,7 @@ function createInjector(modulesToLoad, strictDi) {
       if (!isBoolean(result)) {
         // Workaround for MS Edge.
         // Check https://connect.microsoft.com/IE/Feedback/Details/2211653
-        result = func.$$ngIsClass = /^(?:class\s|constructor\()/.test(Function.prototype.toString.call(func));
+        result = func.$$ngIsClass = /^(?:class\s|constructor\()/.test(stringifyFn(func));
```

Comment:
In the regex, let's replace `\s` with `\b`. Otherwise it doesn't work in the situations like following:
1. `class{ ... }`
2. `class/* foo */ Bar { ... }`